src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -3,8 +3,10 @@ package scalafix.sbt
 import sbt._
 import sbt.Keys._
 import sbt.internal.sbtscalafix.Compat
+import scalafix.internal.sbt.SemanticdbPlugin
 
-/** Command to automatically enable semanticdb-scalac for shell session */
+/** Command to automatically enable semanticdb compiler output for shell session
```

Comment:
This class is getting quite complex as it currently handles:
- different sbt/Zinc versions
  - 1.x with x >= 3
  - 1.x with x < 3
  - 0.13.x
- different scalac versions and how they may support semanticdb
  - 3.x built-in 
  - 2.x with x >= 11 supported by semanticdb-scalac
  - 2.x with x < 11 unsupported by semanticdb-scalac